app/src/navigation/ProfileStackNavigator.tsx

```diff
@@ -1,21 +1,29 @@
 import * as React from "react";
 import { createStackNavigator } from "@react-navigation/stack";
 import ProfileScreen from "./screens/Home";
-import ClaimScreen from "./screens/ClaimScreen";
-import { ClaimType } from "../types/claim";
-import { getClaimFromType } from "../utils/claim-utils";
 import ViewFile from "./screens/ViewFileScreen";
+import NameClaimScreen from "./screens/ClaimScreens/NameClaimScreen";
+import BirthClaimScreen from "./screens/ClaimScreens/BirthClaimScreen";
+import MobileClaimScreen from "./screens/ClaimScreens/MobileClaimScreen";
+import AddressClaimScreen from "./screens/ClaimScreens/AddressClaimScreen";
+import AdultClaimScreen from "./screens/ClaimScreens/AdultClaimScreen";
+import EmailClaimScreen from "./screens/ClaimScreens/EmailClaimScreen";
```

Comment:
May aswell do some gardening...

```suggestion
import AddressClaimScreen from "./screens/ClaimScreens/AddressClaimScreen";
import AdultClaimScreen from "./screens/ClaimScreens/AdultClaimScreen";
import BirthClaimScreen from "./screens/ClaimScreens/BirthClaimScreen";
import EmailClaimScreen from "./screens/ClaimScreens/EmailClaimScreen";
import MobileClaimScreen from "./screens/ClaimScreens/MobileClaimScreen";
import NameClaimScreen from "./screens/ClaimScreens/NameClaimScreen";
```